disperser/metrics.go

```diff
@@ -99,17 +100,18 @@ func (g *Metrics) HandleSuccessfulRequest(quorum string, blobBytes int, method s
 }
 
 // IncrementFailedBlobRequestNum increments the number of failed blob requests
-func (g *Metrics) IncrementFailedBlobRequestNum(quorum string, method string) {
+func (g *Metrics) IncrementFailedBlobRequestNum(statusCode string, quorum string, method string) {
```

Comment:
We should have these http status codes typed somewhere and use that type instead of taking raw string. Something like
```
type HttpStatusCode string
const (
  SuccessHttpStatusCode = "200"
  ...
)
```

Reply:
The HTTP code is already quite standard (probably it's *the* standard)

Reply:
It is. My point isn't whether it's the standard or not. It's that we shouldn't take raw string in this method. 
Why don't we use existing consts defined in standard library? https://go.dev/src/net/http/status.go

Reply:
The same reason why quorum isn't a uint8 which you don't even need a library.